numpy/distutils/fcompiler/pg.py

```diff
@@ -12,7 +12,7 @@ class PGroupFCompiler(FCompiler):
 
     compiler_type = 'pg'
     description = 'Portland Group Fortran Compiler'
```

Comment:
Should we update this description too?

Reply:
`compiler_type` might be also changed, but it will break compatibility for build script that uses `--fcompiler=pg`.
Should we create a new compiler type `nv.py` and leave `pg.py` as is for older versions ?

Reply:
I think that's reasonable - but I think you need to also check `distutils/fcompiler/__init__.py` and add an entry to `tests/test_public_api.py` for the new compiler. There may be other places, worth checking.

Reply:
Thanks for the hints.
I removed the modifications on the `pg.py` files and created the `nv.py` one instead.